fineract-doc/src/docs/en/chapters/architecture/api-performance.adoc

```diff
@@ -0,0 +1,115 @@
+= Introducing Performance Enhancing Read/Write/Batch API Operation Separation
```

Comment:
I'm not really sure why we created a new adoc file and calling it api-performance. Either it should be in the architecture doc or a brand new file which is specific to read/write/batch separation.

Reply:
good point, naming the file is still open for discussion; do you think a file name as api-performance-enhancement or improvement can  have customer documentation on the best practices to work with api to get best performance? cc @edcable.